### PR TITLE
"Last Update" on project card is not correct

### PR DESCRIPTION
**Pull request by RamRamez**
#4204

## Summary by CodeRabbit

- **New Features**
	- Updated project cards to display the latest update creation date instead of the last updated time.

### CODE REVIEW

File: src/apollo/types/types.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ export interface IProject {
 		name?: EProjectStatus;
 	};
 	updatedAt: string;
+	latestUpdateCreationDate: string;
```
Comment (ae2079) [inline src/apollo/types/types.ts:66]: Is it not better to make this field optional? (I think this help safer deploy and prevent crashes)

File: src/apollo/gql/gqlProjects.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ export const PROJECT_CARD_FIELDS = gql`
 			avatar
 		}
 		updatedAt
+		latestUpdateCreationDate
```
Comment (ae2079) [inline src/apollo/gql/gqlProjects.ts:37]: Does the backend serve this field now?

**Review by ae2079 — COMMENTED** [review]
Thanks @RamRamez

**Review by ae2079 — APPROVED** [review]
Looks like the preview link works correctly

File: src/apollo/gql/gqlProjects.ts (hunk 2 above)
Comment (RamRamez) [reply src/apollo/gql/gqlProjects.ts:37]: Yes

File: src/apollo/types/types.ts (hunk 1 above)
Comment (RamRamez) [reply src/apollo/types/types.ts:66]: I have added `latestUpdateCreationDate` to project table, just like updatedAt above that is required field.

Comment (RamRamez) [reply src/apollo/types/types.ts]: I made that optional

**Pull request merged**